src/function/table/table_scan.cpp

```diff
@@ -688,7 +682,7 @@ unique_ptr<GlobalTableFunctionState> TableScanInitGlobal(ClientContext &context,
 }
 
 static unique_ptr<BaseStatistics> TableScanStatistics(ClientContext &context, const FunctionData *bind_data_p,
```

Comment:
If we're going to change this API anyway - let's do `TableScanStatistics(ClientContext &context, TableScanStatisticsInput &input)`